lib/shoryuken/extensions/active_job_adapter.rb

```diff
@@ -61,7 +61,7 @@ def message(queue, job, options = {})
         end
 
         msg[:message_body] = body
-        msg[:message_attributes] = message_attributes
+        msg[:message_attributes] = MESSAGE_ATTRIBUTES.dup
```

Comment:
👍 

Dup-ing a frozen constant seems a lot better than using a memoized constant. This way, the starting point is always the same.